tests/arms_settings_test.py

```diff
@@ -10,42 +11,47 @@ class SampleArmsSettings(ArmsSettings):
 
 
 class TestArmsSettings:
-    def test_arms_settings_are_persisted_upon_creation(self):
+    @staticmethod
+    def test_arms_settings_are_persisted_upon_creation():
         SampleArmsSettings(foo=2, bar="test")
         assert os.path.exists(
             "SampleArmsSettings.json"
         ), "SampleArmsSettings.json was not created upon settings creation"
 
-    def test_arms_settings_are_persisted_upon_modification(self):
+    @staticmethod
+    def test_arms_settings_are_persisted_upon_modification():
         settings = SampleArmsSettings(foo=2, bar="test")
         settings.foo = 4
         settings.bar = "test change"
-        with open("SampleArmsSettings.json") as f:
-            loaded_settings = json.load(f)
+        with open(file="SampleArmsSettings.json", mode="r") as file:
+            loaded_settings = json.load(file)
         assert settings.foo == loaded_settings["foo"]
         assert settings.bar == loaded_settings["bar"]
 
-    def test_arms_settings_can_be_loaded_from_file(self):
+    @staticmethod
+    def test_arms_settings_can_be_loaded_from_file():
         SampleArmsSettings(foo=3, bar="test 2")
         settings = SampleArmsSettings.load()
         assert settings.foo == 3
         assert settings.bar == "test 2"
 
 
 class TestInjectPersistedModelDecorator:
-    def test_inject_persisted_model_decorator_injects_instance_of_arms_settings(self):
+    @staticmethod
+    def test_inject_persisted_model_decorator_injects_instance_of_arms_settings():
         @inject_persistent_models
-        def test_function(ctx, settings: SampleArmsSettings):
+        def _test_function(ctx, settings: SampleArmsSettings):  # pylint: disable=unused-argument
```

Comment:
The pytest think it is a different test because it starts with `test_`

Reply:
even inside a test function?